tasks/main.yml

```diff
@@ -14,7 +14,34 @@
     # it.
     state: latest  # noqa 403
     update_cache: yes
-  when: ansible_os_family == "Debian"
+  when: >
+    ansible_os_family == "Debian" and
+    ansible_distribution_release != "kali-rolling"
+
+- name: Reinstall cryptsetup-initramfs so upgrade doesn't fail
+  apt:
+    name: 'cryptsetup-initramfs'
+    state: latest  # noqa 403
+    update_cache: yes
+  when: ansible_distribution_release == "kali-rolling"
+
+- name: apt-get update
+  shell: apt-get update  # noqa 303 305
+  args:
+    warn: false
+  when: ansible_distribution_release == "kali-rolling"
```

Comment:
Is `ansible_distribution_release` really the only way to tell that this is Kali?  What is `ansible_distribution` for Kali?

Reply:
ansible_distribution are predefined ansible values and Kali is not one of them.